### PR TITLE
 travis ci + flake8 + isort +pytest

### PR DESCRIPTION
**Pull request by Jagrmi-C**
travis ci + flake8 + isort +pytest

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py26, py27, py32, py33, py34, py35, py36, pypy, jython
```
Comment (tbicr) [inline tox.ini]: lets leave py36 only

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[testenv]
+;changedir=tests
+commands = pytest
+deps = pytest
+
+[flake8]
+ignore = D203
+exclude = .git,__pycache__,.tox,docs,*egg,.idea,dist
```
Comment (tbicr) [inline tox.ini]: ignore and exclude can be removed there

File: alembic/versions/b95b832cb0c4_create_user_table.py [hunk 3]
```diff
@@ -27,3 +27,5 @@ def upgrade():
 def downgrade():
     op.drop_table('user')
 
+
+# flake8: noqa
```
Comment (tbicr) [inline alembic/versions/b95b832cb0c4_create_user_table.py]: noqa has no sense there

**Comment by Jagrmi-C** [comment]
changes applied

**Pull request merged**